preview/app/views/pages/index.html.erb

```diff
@@ -155,7 +155,7 @@
           <h2 class="text-h1 font-medium mb-40">Typography</h2>
 
           <p class="ui-text-p1 mb-32">Type definitions do not include margin as it can be contextual. Figma and the below list include recommended margins.</p>
-          <p class="ui-text-p1 mb-32">The below list is created with ui-text-* helpers which include properties beyond the the standard <code class="pi-inline-code">font-size</code> and <code class="pi-inline-code">line-height</code> that text-* includes.</p>
+          <p class="ui-text-p1 mb-32">The below list is created with ui-text-* helpers which include properties beyond the the standard <code class="ui-inline-code">font-size</code> and <code class="ui-inline-code">line-height</code> that text-* includes.</p>
```

Comment:
Is there a reason this class is not replaced in https://github.com/ably/ably-ui/blob/cf8fed962bd3f2341fdf3f6493c21e77793f10cd/preview/app/views/components/icon/_arguments.html.erb

Reply:
nice catch, will update that too 👍🏻

Reply:
Cool! Then that entire css class can be deleted. :)